rmgpy/cantherm/statmech.py

```diff
@@ -200,6 +206,12 @@ def load(self):
         each conformer and appends them to the list of conformers on the
         species object.
         """
+
+        filename, file_extension = os.path.splitext(self.path)
+        if 'yml' in file_extension or 'yaml' in file_extension:
+                self.load_species_from_database = True
+                return
```

Comment:
It seems like you could just call `load_species_from_database_file` here instead of `main.py`, and it would also eliminate the need for the `load_species_from_database` attribute.

Reply:
I agree, will change accordingly